viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -187,8 +187,8 @@ export class Cognite3DViewer {
   /**
    * Gets a list of models currently added to the viewer.
    */
-  public get models(): CogniteModelBase[] {
-    return this._models.slice();
```

Comment:
Should we just change the type of `this._models` to `(Cognite3DModel | CognitePointCloudModel)[]` instead?